packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -63,7 +63,7 @@
  * <p>C) User cancels picking an image. Finish with null result.
  */
 public class ImagePickerDelegate
-    implements PluginRegistry.ActivityResultListener,
+        implements PluginRegistry.ActivityResultListener,
```

Comment:
These white space changes seem to be at odds with our CI's formatting requirements. Please revert.